index.html

```diff
@@ -18,81 +18,82 @@
-              />
-            </a>
-          </li>
-          <li>
-            <a href="#">
-              <img
-                class="social-media-icons"
-                src="images/icons/github.png"
-                alt="Github Icon"
-              />
-            </a>
-          </li>
-          <li>
-            <a href="#">
-              <img
-                class="social-media-icons"
-                src="images/icons/victory.png"
-                alt="Victory Icon"
-              />
-            </a>
-          </li>
-        </ul>
+      <section class="first-section">
+        <section class="header-info">
+          <h1 class="intro">I'm Momanyi <br />Glad to See you!</h1>
+          <p class="about">
+            I’m a software developer! I can help you build a product , feature
+            or website Look through some of my work and experience! If you like
+            what you see and have a project you need coded, don’t hestiate to
+            contact me.
+          </p>
+          <section class="social-media-container">
+            <h4 class="social-media-title">Let's Connect</h4>
+            <ul class="social-media-content">
+              <li>
+                <a href="#">
+                  <img
+                    class="social-media-icons"
+                    src="images/icons/twitter.png"
+                    alt="Twitter Icon"
+                  />
+                </a>
+              </li>
+              <li>
+                <a href="#">
+                  <img
+                    class="social-media-icons"
+                    src="images/icons/linkedin.png"
+                    alt="LinkedIn Icon"
+                  />
+                </a>
+              </li>
+              <li>
+                <a href="#">
+                  <img
+                    class="social-media-icons"
+                    src="images/icons/media.png"
+                    alt="Media Icon"
+                  />
+                </a>
+              </li>
+              <li>
+                <a href="#">
+                  <img
+                    class="social-media-icons"
+                    src="images/icons/github.png"
+                    alt="Github Icon"
+                  />
+                </a>
+              </li>
+              <li>
+                <a href="#">
+                  <img
+                    class="social-media-icons"
+                    src="images/icons/victory.png"
+                    alt="Victory Icon"
+                  />
+                </a>
+              </li>
+            </ul>
+          </section>
+        </section>
       </section>
       <section class="work-card">
         <div class="project-container">
-          <div class="project-img">
-            <a href="#">
-              <img
-                src="images/about-image/tonic-project.png"
-                alt="tonic-project"
-              />
-            </a>
-          </div>
+          <div class="project-img img-3"></div>
```

Comment:
Great design @fullstop125 🤝

- [ ] Your design is a bit different from the design provided in the requirements. You may please try to mimic the Figma design as much as possible. You may increase the size of your images, and the height of your cards, and try to make sure the blank space is not created as the screen size increases.
- Your design
![your card](https://user-images.githubusercontent.com/63917725/178037568-69fe8079-9ac0-42f1-b89b-cec3e75ad36f.PNG)

- Figma design
![Figma card](https://user-images.githubusercontent.com/63917725/178037710-e5136b05-482d-414f-b68b-ff9a57dac462.PNG)

Reply:
resolved